backend/notifications/tests.py

```
@@ -1308,3 +1308,573 @@ def test_clear_removes_reminder_after_verification(self):
+                f'0x{suffix * 40}',
+            )
+            TelegramConnection.objects.create(
+                user=user,
+                platform_user_id=f'11{suffix}',
+                platform_username=f'linked{suffix}',
+                linked_at=timezone.now(),
+            )
+            services.notify('submission.accepted', recipient=user, title='Hi')
+        self.assertEqual(TelegramMessage.objects.count(), 2)
+
+        def send_and_recall_everything(chat_id, text, **kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `send_and_recall_everything`

(ANN202)